src/components/Timestamp/Timestamp.tsx

```diff
@@ -54,6 +64,15 @@ export const Timestamp = ({
 			<time dateTime={isoDateTime.toString()} css={timeStyles}>
 				{timeAgo}
 			</time>
+
+			<span css={anchorStyles} className="comment-permalink-icon">
+				<svg viewBox="0 0 15 15" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I debated on adding this SVG to source but after comparing it to the existing SVG's source has I noticed that the styling is completely different.

Reply:
If this is the paperclip icon, I am still in awe that we use it. I can't see any `link` icon [in source](https://guardian.github.io/source/?path=/docs/packages-source-react-components-icons--playground) - I wonder if it's time to ask if we could have it?

Reply:
For adding SVG's that aren't in source, we can add them into `dotcom-rendering/src/static/icons`. 

Then you can import and use them like: 

```jsx

import MyIcon from '@root/src/static/icons/my-icon.svg'

...

<MyIcon />
```

I think this is the preferred way of adding SVG's in DCR

Reply:
^^ Please ignore the above I mistakenly thought this was a DCR pull request ...

Reply:
Definitely not a blocker either, just wondered if we could work with them on getting something that actually indicates what it is.